Oryx.Cognite/src/Handler.fs

```diff
@@ -119,15 +124,21 @@ module Handler =
         >=> withError decodeError
         >=> json jsonOptions
 
-    let post<'a, 'b, 'c> (content: 'a) (url: string) : HttpHandler<HttpResponseMessage, 'b, 'c> =
+    let postVersion<'a, 'b, 'c> (content: 'a) (url: string) (version: ApiVersion) : HttpHandler<HttpResponseMessage, 'b, 'c> =
         POST
-        >=> setVersion V10
+        >=> setVersion version
         >=> setResource url
         >=> setContent (fun () -> new JsonPushStreamContent<'a>(content, jsonOptions) :> _)
         >=> fetch
         >=> withError decodeError
         >=> json jsonOptions
 
+    let post<'a, 'b, 'c> (content: 'a) (url: string) : HttpHandler<HttpResponseMessage, 'b, 'c> =
```

Comment:
Use `>=>` composition instead of passing version as a parameter. E.g use `setVersion V10 >=> post`. Then have a `postV10` for pinning to a concrete version.